CriticalMapsUITests/CriticalMapsUITests.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  CriticalMapsUITests.swift
+//  CriticalMapsUITests
+//
+//  Created by Felizia Bernutz on 06.10.19.
+//  Copyright © 2019 Pokus Labs. All rights reserved.
+//
+
+import XCTest
+
+class CriticalMapsUITests: XCTestCase {
+
+    override func setUp() {
+        super.setUp()
+        continueAfterFailure = false
+
+        let app = XCUIApplication()
+        app.launchArguments += [
+            "SKIP_ANIMATIONS"
+        ]
+        setupSnapshot(app, waitForAnimations: false)
+        app.launch()
+
+        addUIInterruptionMonitor(withDescription: "Allow location permissions") { alert in
+            alert.buttons["Allow While Using App"].tap()
+            return true
+        }
+        app.tap()
+    }
+
+    func testScreenshots() {
+        let app = XCUIApplication()
+        snapshot("01_map")
+
+        let closeButton = app.navigationBars.buttons.element(boundBy: 0).firstMatch
+        let backButton = closeButton
+
+        XCTContext.runActivity(named: "Show Chat") { _ in
```

Comment:
🤯 Didn't know about that API

Reply:
I learned about it some months ago and it makes the test report in Xcode so much more structured. Even the code! I always used comments to mark different use cases, but this is great for readability as well 👍